src/Microsoft.AspNetCore.SignalR.Client.Core/Internal/SignalRClientLoggerExtensions.cs

```diff
@@ -40,44 +40,53 @@ internal static class SignalRClientLoggerExtensions
         private static readonly Action<ILogger, string, Exception> _dropStreamMessage =
             LoggerMessage.Define<string>(LogLevel.Warning, new EventId(9, nameof(DropStreamMessage)), "Dropped unsolicited StreamItem message for invocation '{invocationId}'.");
 
+        private static readonly Action<ILogger, string, Exception> _dropStreamCompletionMessage =
```

Comment:
This is always a bad idea. You can't add logging events in between. Just add it at the end. We're going to have to do that once we ship anyways.

Reply:
OK - I thought it was fine because we did not ship anything yet.. . How about deleting messages? Should we leave the gap?

Reply:
I'm ok fixing it now (you already did it), we shouldn't do it after we ship.